client/src/app/actions/apiRoute.js

```diff
@@ -0,0 +1,9 @@
+//For production
```

Comment:
What is this file used for?

Reply:
it was to protect my api logic but i am now using the .env file